src/main-oe/sgxlkl_run_oe.c

```diff
@@ -331,14 +335,20 @@ static void help_enclave_config()
     }
 }
 
-#if DEBUG && VIRTIO_TEST_HOOK
-/* Control the event channel notification between host & guest */
-
-/* Signal handler to resume paused evt chn to resume guest */
+#ifdef DEBUG
+// Signal handler for thread stack dumps and virtio event channel tests
 static void sgxlkl_loader_signal_handler(int signo)
 {
     switch (signo)
     {
+        case SIGTERM:
```

Comment:
Can we do this for SIGUSR1 as well?  It would be convenient to have a way of doing this form outside without stopping the program.

Reply:
Ok, that's an easy change to make (especially since the stack traces always come from the scheduler state and do not interrupt thread execution).